src/snowflake/snowpark/stored_procedure.py

```diff
@@ -79,9 +88,15 @@ def __init__(
         self._execute_as = execute_as
         self._anonymous_sp_sql = anonymous_sp_sql
         self._is_return_table = isinstance(return_type, StructType)
-
         self._packages = packages
 
+        # If None, no ast will be emitted. Else, passed whenever sproc is invoked.
+        self._ast = _ast
+        self._ast_id = _ast_id
+        self._stmt = (
```

Comment:
nit: you can move the comment to the line above and do
```
self._stmt = _stmt
```

Reply:
done!